packages/gatsby-image/src/index.js

```diff
@@ -332,24 +430,24 @@ class Image extends React.Component {
 
           {/* Show the blurry base64 image. */}
           {image.base64 && (
-            <Img src={image.base64} {...placeholderImageProps} />
+            <picture>
```

Comment:
Yeah, I'm curious if the internal structure of this package is considered public API. It would be fairly straightforward to switch based on `if (fixedImages || fluidImages)`, but it may look a bit messier.

Reply:
If the core devs are ok with it, I think it's fine for the most part. Lock files should avoid the few that are likely to be affected with existing projects.

Reply:
oh yeah I agree with @polarathene we need to fix this.

`if (imageVariants) { /* picture logic */ } else { /* old behaviour */ }`

Reply:
Github has some weird way of showing updates, it looked like you gave a new comment, rather than replying to an old review comment :confused: 

I can't say how many projects would be impacted by this, but another option is just to bump the major version of the package? (although due to concerns with the last PR I contributed and how this package version management is handled, that's not a great idea)